gumrya-web/src/test/java/gumrya/service/UserServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package gumrya.service;
+
+import gumrya.dto.UserRequest;
+import gumrya.dto.UserResponse;
+import gumrya.exception.DuplicateException;
+import gumrya.exception.NotFoundException;
+import gumrya.user.domain.model.User;
+import gumrya.user.domain.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+@DisplayName("User 비지니스 흐름 테스트")
+class UserServiceTest {
+
+    @InjectMocks
+    private UserService userService;
+
+    @Mock
+    private UserRepository userRepository;
+
+    private User user;
+    private UserRequest userRequest;
+
+    @BeforeEach
+    void setup() {
+        this.userRequest = new UserRequest(1L, "ryan@gmail.com", "12342n1k3l", "검프", "나는 검프다", "www.url.com",
+                "github" +
+                        ".com/younghoonkwon");
```

Comment:
```suggestion
                "github" +
                        ".com/younghoonkwon");
```
라이언만의 이유가 있는 줄바꿈일까요??!